uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1017.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1017 {
+
+  /**
+   * Migrating Phone Types and Deleting unwanted Phone Types
+   */
+  public function upgrade_1017() {
+    $this->migratePhoneTypes();
+    $this->relabelPhoneType("phone", "Landline");
+    $this->reservePhoneTypes(["phone", "mobile"]);
```

Comment:
Shouldn't this be `$this->reservePhoneTypes(['Landline', 'Mobile']);` ? since Phone has been relabelled to Landline above.  I understand that it was the label that was changed but I think the name should be changed also to be consistent with the other phone types that have same name and label.

Reply:
Agreed, but i will ask Mike about changing the name also.